dist/reactTemplates.js

```diff
@@ -243,12 +243,16 @@ function generateProps(node, context) {
             props[propKey] = convertText(node, context, val.trim());
         }
 
+        // Note: The correct way to handle require being output in generated
+        // templates is to ensure the output is processed by webpack.
         if (node.name === 'img' && propKey === 'src') {
-            let evaluated = props[propKey];
-            if (/^"\/?bhf-assets/.test(evaluated)) {
-                evaluated = 'require(' + evaluated.replace(/^"\/?bhf-assets/, '"bhf-assets') + ')';
+            let nv = val.trim();
+            // Remove any leading / so module aliases are easier, the
+            // code should probably be updated but this is currently easier.
+            if (nv[0] === '/') {
+                nv = nv.slice(1);
             }
-            props[propKey] = evaluated;
+            props[propKey] = `require('${ nv }')`;
         }
```

Comment:
The existing implementation will only work on src with `/bhf-assets/...` or `bhf-assets/...`.

My concern here is, what if we have a an img src with `https://www.example.com/photo.jpg`?

Do we have img src's hosted at other 1st level paths? Development-wise, we are pointing references to img's and background url-s with `bhf-assets`.